notebook/static-src/notebook/js/celltoolbarpresets/rawcell.js

```diff
@@ -72,15 +69,13 @@ define([
       },
       // name
       "Raw NBConvert Format"
-  );
+    );
 
-  var register = function (notebook) {
+    var register = function (notebook) {
     CellToolbar.register_callback('raw_cell.select', select_type, ['raw']);
     raw_cell_preset.push('raw_cell.select');
 
     CellToolbar.register_preset('Raw Cell Format', raw_cell_preset, notebook);
     console.log('Raw Cell Format toolbar preset loaded.');
-  };
-  return {'register': register};
-
-});
+    };
+    exports.register = register;
```

Comment:
No - it's a (erroneous) part of re-indenting things I unindented, to drop the changed lines from 15k to 1k.  This helps tremendously with rebases.  I'll open a PR after this to fix the indenture.